packages/office-ui-fabric-react/src/components/Calendar/CalendarMonth.tsx

```diff
@@ -168,6 +169,7 @@ export class CalendarMonth extends BaseComponent<ICalendarMonthProps, ICalendarM
                     ? strings.nextYearAriaLabel + ' ' + dateTimeFormatter.formatYear(addYears(navigatedDate, 1))
                     : undefined
                 }
+                title={strings.nextYearAriaLabel ? strings.nextYearAriaLabel : undefined}
```

Comment:
The typing already constrains this type to `string | undefined`, so I think the ternary check is redundant.

Reply:
I agree, I updated the PR with the ternary removed.